lib/rucio/rse/protocols/webdav.py

```diff
@@ -516,6 +517,7 @@ def stat(self, path):
             if file.size is None:
                 continue
 
+            dict_['adler32'] = adler32(file)
```

Comment:
This won't work. This is a remote file, you can't (locally) adler32 checksum it. 

The idea of the issue is to really to check if adler32 is mentioned in the propfind, which I doubt. In that case we just remove it from the documented line in 488.

Reply:
So I didn't found adler32 too and I removed its mentioning in docstring. 

It is what I was expected to do or I can make other changes here?